src/ClassApplication/Controllers/ClassController.cs

```diff
@@ -40,11 +41,89 @@ public async Task<ActionResult<string>> CreateClassroom(Classroom classroom)
         /// <param name="id">classroom id</param>
         /// <returns>An array containing a value determined by the parameter</returns>
         [HttpGet("GetClassByID/{id}")]
-        public async Task<ActionResult<string>> GetClassroom(string id)
+        /*
+                public async Task<ActionResult<string>> GetClassroom(string id)
+                {
+                    //to check if i am pushing it right
+                    //throw new NotImplementedException();
+         }
+         */
+        public async Task<ActionResult<string>> GetClassroomByID(string id)
         {
-            throw new NotImplementedException();
+
+
+            string Text = " SELECT * FROM c WHERE c.noteId =@id";
+            QueryDefinition query = new QueryDefinition(Text).WithParameter("@Id", id);
+
+            FeedIterator<Classroom> iterator = classesContainer.ReadItemAsync<Classroom>(query);
+
+            List<Classroom> classroomId = new List<Classroom>(classId);
+
+            //this is an example to add class
+            classroomId.Add("CS 3364"); //example
+            classroomId.Add(new (ClassController) { GetClassroom = "CS 3375" });
+            var classId = id;
+
+            while (iterator.HasMoreResults)
+            {
+                FeedResponse<Classroom> result = await iterator.ReadNextAsync();
+                foreach (Classroom classId in result)
+                {
+                    classroomId.Add(classId);
+                }
+            }
+            //Returns data from a classroom from a given classroom ID
+            return classroomId;
+
+
+            if (classroomId == null)
+            {
+                Console.WriteLine("No any Classes. Click '+' and type new class Id to create one.")

```

Comment:
So as far as logging and getting user input, that will be handle on the front end, so we do not need to write anything to console or grab user input...

Reply:
Got It. Thank you very much.